.travis.yml

```diff
@@ -21,8 +21,8 @@ env:
   - secure: eNfbQI7Pkp3mjTFQgjrVVldmyE/Yn3+iCMVykhXppC85dJaCFMpjwPUfZa7p38tnnH7jhng2JL2tAXnJ4uOry0B3m9xYR7fNYXg/9uj1988FHvpDhfDFEKcMF8s70mr11HRijj6U8p6WDt/0ygD6aZ0LJzWzxhMFecFavOb6XOw=
   matrix:
   - LINT_CHECK="1"
-  - TESTS="1" ODOO_REPO="odoo/odoo"
-  - TESTS="1" ODOO_REPO="OCA/OCB"
+  - TESTS="1" ODOO_REPO="odoo/odoo" EXCLUDE="website_wow"
+  - TESTS="1" ODOO_REPO="OCA/OCB" EXCLUDE="website_wow"
```

Comment:
Note I think this is a bit cleaner, but I think it mean that you'll have to actually install this module on Runbot vs. it being pre-installed. I'm indifferent on whether this commit should stay, and am interested in others' opinions here